swagger.yaml

```diff
@@ -99,24 +100,32 @@ paths:
       tags:
         - public
       summary: calculate ckmeans over a given category and geography type
-      descriptions: Returns a list of the maximum values in each ckmean cluster.
+      description: |
+        Uses ckmeans algorithm to estimate natural breakpoints in the given census data category (*cat* parameter), 
+        filtered to only rows for the specified geography type (*geotype* parameter). Returns a list of the maximum 
+        values (i.e. the upper breakpoint) in each ckmean cluster.
       parameters:
         - in: path
           name: year
-          description: census year
+          description: Census year (currently only 2011 available).
```

Comment:
If we get synthetic data for 2021 quickly this might not be true in a week or two!